pkgs/tools/system/dool/default.nix

```
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace dool \
       --replace \
```

```suggestion
      --replace-fail \
```
This will catch any future changes.

---

I have applied your suggestion, thanks for the review.